packages/component-wrapper/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should all these `component-wrapper` changes be here? Or are you trying to PR into the wrong base branch?

Reply:
I merged the component-wrapper changes down into this branch, but then I realised i'd missed the .babelrc file out so it failed the check